shared/django_apps/rollouts/migrations/0001_initial.py

```diff
@@ -0,0 +1,155 @@
+# Generated by Django 4.2.7 on 2024-02-19 23:18
+
+import django.contrib.postgres.fields
+import django.db.models.deletion
+from django.db import migrations, models
+
+import shared.django_apps.rollouts.models
+
+
+class Migration(migrations.Migration):
```

Comment:
Could you consolidate these 3 migrations into 1 migration for cleanliness? You'd basically need to 

1) delete these individual migration files
2) delete the migrations from your db entries - the `django_migrations` db table
3) remake this migration with `python manage.py makemigrations` - you can specify the app here if you want

Could you also mimic this behavior of adding the SQL that this migration would output? Sample [here](https://github.com/codecov/codecov-api/blob/5111f78d329225b7b9189feab78d4dd65335754f/codecov_auth/migrations/0051_user_customer_intent.py#L11), you can run that by doing `python manage.py sqlmigrate <app_name> <app_number>` , so for you i'd be `python manage.py sqlmigrate rollouts 0001`

Lmk if you have any questions 👌

Reply:
yeah I could combine them into one.

And yeah I can add the raw SQL to the migration as comment. Thanks